mlflow/pipelines/steps/transform.py

```diff
@@ -127,13 +127,11 @@ def _build_profiles_and_card(
 
         if not self.skip_data_profiling:
             # Tab 1 and 2: build profiles for train_transformed, validation_transformed
-            train_transformed_profile = get_pandas_data_profile(
-                train_transformed,
-                "Profile of Train Transformed Dataset",
+            train_transformed_profile = get_pandas_data_profiling_comparison_view(
+                [["Profile of Train Transformed Dataset", train_transformed]]
             )
-            validation_transformed_profile = get_pandas_data_profile(
-                validation_transformed,
-                "Profile of Validation Transformed Dataset",
+            validation_transformed_profile = get_pandas_data_profiling_comparison_view(
+                [["Profile of Validation Transformed Dataset", validation_transformed]]
             )
```

Comment:
Shouldn't these 2 profiles be compared in a single tab?

Reply:
I was waiting for @mengxr comment to confirm if we like to do the same for transform step as well.